proiect/src/order/OrderStatus.java

```diff
@@ -0,0 +1,5 @@
+package order;
+
+public enum OrderStatus {
+	Sent, Preparing, AwaitingPickUp, Delivering, Arrived, Done, Cancelled
```

Comment:
usually we use UPPERCASE for constants and Enum values

Reply:
Resolved in issue #6